code/modules/mob/living/simple_animal/hostile/mining_mobs/drakeling.dm

```diff
@@ -147,7 +156,7 @@
 	button_icon = 'icons/obj/wizard.dmi'
 	button_icon_state = "fireball"
 	cooldown_time = 1.6 SECONDS //kinetic gun
-	active_msg = span_notice("You prepare %YOUR fire breath attack")
+	active_msg = span_notice("You prepare the fire breath attack")
```

Comment:
There's another one of this in ``/datum/action/cooldown/spell/pointed/drakeling``